pkgs.sh

```diff
@@ -85,4 +86,5 @@ export PATCH_COREUTILS_CHMOD=https://www.linuxfromscratch.org/patches/lfs/11.1/c
 export PATCH_GLIBC=https://www.linuxfromscratch.org/patches/lfs/11.1/glibc-2.35-fhs-1.patch
 export PATCH_KBD=https://www.linuxfromscratch.org/patches/lfs/11.1/kbd-2.4.0-backspace-1.patch
 export PATCH_PERL=https://www.linuxfromscratch.org/patches/lfs/11.1/perl-5.34.0-upstream_fixes-1.patch
+export PATCH_SYSLINUX=https://gitweb.gentoo.org/repo/gentoo.git/plain/sys-boot/syslinux/files/syslinux-6.03-sysmacros.patch
```

Comment:
So I understand syslinux is some kind of bootloader. Will it "replace" GRUB whenever the script builds an ISO? Or will they need to coexist?

Reply:
Grub will operate a boot of `vmz` and `imglinux` files. Syslinux helps to boot them from `fat` systems on which `iso` operates and boots.

Reply:
This means that they will cooperate...
But RN i have some issues with building `syslinux`... Must check how it built from that project.